vm-setup/roles/v1aX_integration_test/templates/cluster-template-cluster.yaml

```diff
@@ -13,19 +13,19 @@ spec:
 {% if CAPM3_VERSION == "v1alpha4" %}
     apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
 {% else %}
-    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha5
```

Comment:
```suggestion
    apiVersion: infrastructure.cluster.x-k8s.io/{{CAPM3_VERSION}} 
```

Reply:
Agree! And we can get rid of the if/else thing, right?

Reply:
Yes will update the PR.

Reply:
Done